### ISSUE TITLE
No link from CVE package table to package

### ISSUE THREAD

**Issue by ctron** [trustification/trustify-ui#197]
Having the details of a package listed in a CVE, there's no way to link to the package:

![image](https://github.com/user-attachments/assets/c1ed7682-a67f-4255-bebd-3ae317fb4306)

One needs to remember (because it is not even possible to copy the button label) and search for this in the packages view. Which might turn up other matches, as it is a sub-string search.

**Comment by kahboom** [comment]
I think we are doing away with this drawer view altogether and reverting back to links to the package detail page, but @carlosthe19916 could you please confirm?

**Comment by carlosthe19916** [comment]
Yes, the Drawer was removed therefore this issue can not be reproduced any longer. Closing it
